test/functional/fixtures/fake-lsp-server.lua

```diff
@@ -789,15 +789,19 @@ function tests.code_action_with_resolve()
     end,
     body = function()
       notify('start')
-      local cmd = {
-        title = 'Command 1',
-        command = 'dummy1',
-      }
+      local cmd = { title = 'Action 1' }
       expect_request('textDocument/codeAction', function()
         return nil, { cmd }
       end)
       expect_request('codeAction/resolve', function()
-        return nil, cmd
+        return nil,
```

Comment:
This test was incorrect. According to the spec, only code actions should be resolved (both the argument and result of the request should be a `CodeAction`, not a `Command`).